CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# HomeUniteUs Contributing Guide
+
+To begin working on the `HomeUniteUs` project you will need to be an active member of Hack for LA. If you have not completed the onboarding process, then please prefer to the [Getting Started Page](https://www.hackforla.org/getting-started) for full instructions. This onboarding will provide you access to the Hack for LA Slack, GitHub, and Google Drive.
+
+Once you have access to these systems you can introduce yourself on our [Slack channel](https://hackforla.slack.com/archives/CRWUG7X0C) and check for open positions by reviewing the [development job board](https://github.com/hackforla/engineering/projects/2).
+
+Welcome to Hack for LA, and thank you for contributing 🥳🥳!!
+
+## First Steps
+
+Once you've completed onboarding and been assigned a role on the team, you can begin reviewing our internal documentation and open tasks.
+
+1. Join the [#home-unite-us slack channel](https://hackforla.slack.com/archives/CRWUG7X0C).
+2. Request access to, and review the documents in, the [home-unite-us shared drive](https://drive.google.com/drive/u/0/folders/1ahxiD9rIsBtx0yAPlPcPaGw8zGrfHHm9).
+3. Visit our [project boards](https://github.com/hackforla/HomeUniteUs/projects) and see issues in the prioritized backlog.
+
+## Code of Conduct
+
+As part of Hack for LA, we are passionate about the organization's [Code of Conduct](https://github.com/hackforla/codeofconduct). Please help us uphold these policies so that we can continue to maintain an open, safe, and inclusive culture. It is everyone's responsibility to maintain these policies, so please report any violations to the [Hack for LA team](https://github.com/hackforla/codeofconduct#email-template-for-anti-harassment-reporting).
+
+## Issues
+
+### Create a new issue
+
+Before creating a new issue, search through the [current list of issues](https://github.com/hackforla/HomeUniteUs/issues) to determine if the issue has been identified. If it is already present, then feel free to comment on the existing issue to confirm the problem and add any additional information.
+
+To create a new issue, please use the blank issue template (available when you click New Issue). Issues with more information and fuller descriptions are more likely get addressed, so please include as much information as possible.
```

Comment:
should we give a template on what info should include or is it better to leave open-ended? Are there any labels that are required such as story points etc.?

Example copied the below from the website team:
### Overview
REPLACE THIS TEXT - Clearly state the purpose of this issue in 2 sentences or less.  We write ours a modified user story in this format: _We need to do X for Y reason._

### Action Items
REPLACE THIS TEXT - List the research to be done, or the steps to be completed.
Note: If the steps can be divided into tasks for more than one person, we recommend dividing it up into separate issues, or assigning it as a pair programming task.

### Resources/Instructions
REPLACE THIS TEXT - Provide links to resources or instructions that may help with this issue. This can include files to be worked on, external sites with solutions, documentation, etc.

Reply:
That's right. Template shows by default and is stored in the repo. I added a link to the template in [ebfba50](https://github.com/hackforla/HomeUniteUs/pull/557/commits/ebfba503f84ed549893cd2f68623dc90597fdafe).

The template is identical to what @agosmou pasted.